.gitignore

```diff
@@ -3,12 +3,11 @@ build-*/
 code/
 downloads/
 artifact/
-*.exe
-*.dll
-*.s
-*.gkd
-*.o
-*.pdb
-*.dump
-*.log
-*.a
+tests/*.exe
```

Comment:
maybe it still makes sense to exclude everything in the build folder by default with one rule and disable this rule if it is needed for some debugging.

Reply:
The build folder is ignored entirely, see line 2 of this file.